valtimo-implementation/components/valtimo-list.md

```diff
@@ -0,0 +1,101 @@
+# Valtimo list

```

Comment:
Please follow the styling guide found [here](https://docs.valtimo.nl/readme/style-guide).

Additionally, as per [this page](https://docs.valtimo.nl/readme/contributing#writing-documentation-related-to-an-upcoming-release), make sure to link to this page on the appropriate pages (e.g. include it in SUMMARY.md).